db/seeds.rb

```diff
@@ -12,3 +12,30 @@
 end
 
 Rails.logger.debug "Personas successfully created!"
+
+Rake::Task["gias_update"].invoke unless GiasSchool.any?
+gias_scools = GiasSchool.last(SCHOOLS.count)
+
+SCHOOLS.each.with_index do |school_attributes, index|
```

Comment:
For what it's worth, I think the `PERSONAS` constant is more meaningful as a global constant because it is also used for validation in the `Persona` model:

https://github.com/DFE-Digital/itt-mentor-services/blob/a485beef8f5ddedbf32f3484d749162214958fed/app/models/persona.rb#L18-L21

However I think `SCHOOLS` only serves a purpose specifically within this `seeds.rb` file, and doesn't need to be used elsewhere. So I feel like it makes the most sense for it to be located directly where it's used – i.e. in the same file.